packages/extension-api/src/extension-api.d.ts

```diff
@@ -4319,6 +4321,13 @@ declare module '@podman-desktop/api' {
      */
     version: string;
     path: string;
+
+    /**
+     * How the cli tool has been installed
+     * - user: it has been installed by the user externally from podman desktop. Its update process is disable.
+     * - app: it has been installed through podman desktop. It can be updated
+     */
+    installationSource?: CliToolInstallationSource;
```

Comment:
hello, I'm not sure here to follow.

>
> * How the cli tool has been installed
>     * - user: it has been installed by the user externally from podman desktop. Its update process is disable.
>     * - app: it has been installed through podman desktop. It can be updated

it seems it's a flag provided by the extension, so how does it work ? an extension can tell it has been installed by Podman Desktop while it has been installed by the user

maybe just the wording. Rather than 'podman desktop' it seems it's the 'extension'

Reply:
I added it bc i thought that an extension could download the binary in a specific place and podman desktop could not know it. So an extension is responsible to tell if it has been installed by them or not. If not specified, the default value is user/external so it cannot be updated.

Reply:
yes wording, as it's managed by the extension, not Podman Desktop.